test/integration/integration_test.go

```diff
@@ -43,7 +45,7 @@ const (
 	// scripts are made available for integration testing.
 	//
 	// Port 23456 must match the port used in the fake image Dockerfiles
-	FakeScriptsHttpURL = "http://127.0.0.1:23456/sti-fake/.sti/bin"
+	FakeScriptsHTTPURL = "http://127.0.0.1:23456/sti-fake/.sti/bin"
```

Comment:
i'll defer to @soltysh's eyes on this one, but HTTP would probably have been my instinct too :)